modules/command-engine/core/src/main/scala/surge/internal/persistence/PersistentActor.scala

```diff
@@ -165,13 +163,7 @@ class PersistentActor[S, M, R, E](
 
   private case class EventPublishTimedOut(reason: Throwable, startTime: Instant) extends Internal
 
-  protected case class InternalSpans(messageProcessingSpan: Option[Span] = None)
```

Comment:
removing these internal spans from here for now because they were getting in my way